packages/app/src/app/floating-ui/floating-ui.component.ts

```diff
@@ -95,6 +95,7 @@ export class FloatingUIComponent {
     }
     this.logo = this.session.getLogo() ?? '';
     this.workLocal = this.session.isWorkLocal();
+    this.state.observeIsReadOnly().pipe(takeUntil(this._ngUnsubscribe)).subscribe(this.isReadOnly);
```

Comment:
i would suggest to assign the property directly, isReadonly is only used in the template, so then we wouldn't even need the take until

i.e.
```ts
public isReadOnly = this.state.observeIsReadOnly();
```